function handlePromise() {}

const demo = async () => {
  var _a;

  try {
    _a = await handlePromise();
  } catch (e) {
    console.log("handlePromise catch error----", e);
  }

  return a;
};

async function demo2() {
  var _b;

  try {
    _b = await handlePromise();
  } catch (e) {
    console.log("handlePromise catch error----", e);
  }

  return b;
}